src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
The only other compromise that I could think of is that this patch could have some mechanism for not including `/usr/share` in the RPP (by an environment variable maybe) and we could trigger that mechanism when using ROS from `/opt`.